### PR TITLE
Update part of examples written in CSharp

### PR DESCRIPTION
**Pull request by eugene-yakush**
This PR removes unnecessary console output and exception handling in the following samples:

- ChromiumBinariesResolver.Wpf
- ContextMenu.SpellCheck.WinForms
- ContextMenu.SpellCheck.Wpf
- ContextMenu.WinForms
- ContextMenu.Wpf
- CreateHtmlUi.Wpf
- CustomShortcuts.WinForms
- Dom.DragAndDrop.WinForms
- DragAndDrop.Wpf
- JavaScriptBridge.WinForms
- KeyboardEventSimulation.WinForms
- KeyboardEventSimulation.Wpf
- Kiosk.Wpf
- MailToHandling.WinForms
- TransparentWebPage.Wpf
- UiAutomation.Wpf
- Zoom.Wpf

### CODE REVIEW

File: csharp/DragAndDrop.Wpf/MainWindow.xaml.cs [hunk 1]
```diff
@@ -76,31 +76,31 @@ private void ExtractData(string name, IDataObject dataObject)
 
             StringBuilder sb =
                 new StringBuilder("=====================================================");
-            sb.AppendLine("\nEvent name:" + name);
-            sb.AppendLine("\nIDataObject:" + dataObject);
+            sb.AppendLine($"\nEvent name:{name}");
```
Comment (vladislav-tsvek) [inline csharp/DragAndDrop.Wpf/MainWindow.xaml.cs:79]: It is necessary to add the space after colons in this file.
For ex.:
```suggestion
            sb.AppendLine($"\nEvent name: {name}");
```

File: csharp/KeyboardEventSimulation.Wpf/MainWindow.xaml.cs [hunk 2]
```diff
@@ -45,40 +45,35 @@ public partial class MainWindow : Window
...
-                                                           }
-                                                          .Build());
+            Task.Run(() =>
+                 {
+                     engine = EngineFactory.Create(new EngineOptions.Builder
+                                                       {
+                                                           RenderingMode =
+                                                               RenderingMode.OffScreen
+                                                       }
+                                                      .Build());
```
Comment (vladislav-tsvek) [inline csharp/KeyboardEventSimulation.Wpf/MainWindow.xaml.cs:55]:
```suggestion
                     {
                         RenderingMode = RenderingMode.OffScreen
                     }.Build());
```

File: csharp/Kiosk.Wpf/MainWindow.xaml.cs [hunk 3]
```diff
@@ -40,33 +40,28 @@ public partial class MainWindow : Window
...
-                                                          .Build());
+            Task.Run(() =>
+                 {
+                     engine = EngineFactory.Create(new EngineOptions.Builder
+                                                       {
+                                                           RenderingMode =
+                                                               RenderingMode
+                                                                  .HardwareAccelerated
+                                                       }
+                                                      .Build());
```
Comment (vladislav-tsvek) [inline csharp/Kiosk.Wpf/MainWindow.xaml.cs:51]:
```suggestion
                     {
                         RenderingMode = RenderingMode.HardwareAccelerated
                     }.Build());
```

File: csharp/ChromiumBinariesResolver.Wpf/MainWindow.xaml.cs [hunk 4]
```diff
@@ -30,9 +30,6 @@
 
 namespace ChromiumBinariesResolver.Wpf
 {
-    /// <summary>
-    ///     Interaction logic for MainWindow.xaml
-    /// </summary>
     public partial class MainWindow : INotifyPropertyChanged
```
Comment (vladislav-tsvek) [inline csharp/ChromiumBinariesResolver.Wpf/MainWindow.xaml.cs:33]:
```suggestion
    /// <summary>
    ///     This example demonstrates how to deploy Chromium binaries
    ///     over network.
    /// </summary>
    public partial class MainWindow : INotifyPropertyChanged
```

**Review by vladislav-tsvek — CHANGES_REQUESTED** [review]
Please take a look at the comments.

**Review by vladislav-tsvek — APPROVED** [review]
LGTM

**Review by anna-dolbina — APPROVED** [review]

**Pull request merged**